docs/guide/newReplacement.md

```diff
@@ -84,17 +89,18 @@ The config is consists of 2 parts, Modules and Definitions
 
 ##### Modules
 The `modules` section represents the code modifications which should be performed.
-top level it contains a custom grouping id, it can be freely chosen, e.g. `jQuery Function Extensions`
-the second level contains a unique id for the replacement, it can also be freely chosen, e.g. `*.cursorPos`
- Keep in mind that the configName is passed, e.g. to the `Finder` and used there, so it could play rule
+Top level it contains a custom grouping id, which can be freely chosen, e.g. `jQuery Function Extensions`
+The second level contains a unique id for the replacement, it can also be freely chosen, e.g. `*.cursorPos`,
+unless a `Finder` uses it for determining a name. e.g. 
```

Comment:
I think the example is missing from the position after "e.g."

Reply:
made it clearer